homeassistant/components/generic/config_flow.py

```diff
@@ -93,6 +97,14 @@ def build_schema(user_input):
             CONF_VERIFY_SSL, default=user_input.get(CONF_VERIFY_SSL, True)
         ): bool,
     }
+    if is_options_flow:
+        spec = {
+            **spec,
+            vol.Required(
+                CONF_LIMIT_REFETCH_TO_URL_CHANGE,
+                default=user_input.get(CONF_LIMIT_REFETCH_TO_URL_CHANGE, False),
+            ): bool,
+        }
```

Comment:
```suggestion
        spec[vol.Required(
                CONF_LIMIT_REFETCH_TO_URL_CHANGE,
                default=user_input.get(CONF_LIMIT_REFETCH_TO_URL_CHANGE, False),
        )] = bool
```

Black will likely complain about this suggestion

Reply:
I copied the approach from what Frenck did on the mjpeg camera integration

Reply:
While it works fine, copying the whole dict is a bit inefficient.

Reply:
Ok, done eb3936e